DependencyInjection/Compiler/EntityListenerPass.php

```diff
@@ -62,10 +66,6 @@ public function process(ContainerBuilder $container)
                 if (! isset($attributes['lazy']) && $resolverSupportsLazyListeners || $lazyByAttribute) {
                     $listener = $container->findDefinition($id);
 
-                    if ($listener->isAbstract()) {
-                        throw new InvalidArgumentException(sprintf('The service "%s" must not be abstract as this entity listener is lazy-loaded.', $id));
```

Comment:
This check is done from within `findAndSortTaggedServices` now, so no need for it here anymore